examples/console/main.go

```diff
@@ -109,8 +113,10 @@ func run() error {
 				fmt.Fprintf(os.Stderr, "agent channel opened\n")
 			case agentpkg.PaymentReceivedEvent:
 				fmt.Fprintf(os.Stderr, "agent channel received payment: iteration=%d balance=%s\n", e.CloseAgreement.Envelope.Details.IterationNumber, amount.StringFromInt64(e.CloseAgreement.Envelope.Details.Balance))
+				closeAgreements = append(closeAgreements, e.CloseAgreement)
 			case agentpkg.PaymentSentEvent:
 				fmt.Fprintf(os.Stderr, "agent channel sent payment and other participant confirmed: iteration=%d balance=%s\n", e.CloseAgreement.Envelope.Details.IterationNumber, amount.StringFromInt64(e.CloseAgreement.Envelope.Details.Balance))
+				closeAgreements = append(closeAgreements, e.CloseAgreement)
```

Comment:
❓  will this add the same close agreement added a couple lines up? If so what if we only include this line, so that only fully signed close agreements are stored?

Reply:
The sender only gets the `PaymentSentEvent` after the payment is confirmed by the other side, and the receiver only gets the `PaymentReceivedEvent`, so there won't be a duplicate.

Reply:
ah right 👍